src/server/models/frame.js

```diff
@@ -5,6 +5,11 @@ class Frame {
             throw new Error("Invalid data provided");
         }
 
+        //in case of null or undefined input, we set to 0
+        if (c1 == null){c1 = 0;}
```

Comment:
Prefer removing braces and return a line

Reply:
doen